src/main/java/de/unistuttgart/overworldbackend/data/PlayerTeleporterData.java

```diff
@@ -0,0 +1,27 @@
+package de.unistuttgart.overworldbackend.data;
+
+import javax.validation.constraints.NotNull;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.FieldDefaults;
+
+/**
+ * Data Transfer Object to communicate a change of a teleporter for a player
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class PlayerTeleporterData {
+
+    @NotNull
+    int index;
```

Comment:
I don't think `index` is a good idea.
I prefer letting the frontend send a string, as that will be far more backwards compatible.

Reply:
Also: Why `@NotNull`? And why suddenly `javax.validation.constraints` instead of `org.springframework.lang`?

Reply:
After consultation with @PatrickHoltz , we don't think the idea is so good, because we have made it uniform as an index so far and the frontend team gets along so well.

Reply:
Okay, fine.